src/Provider/FileProvider.php

```diff
@@ -257,9 +257,11 @@ public function getDownloadResponse(MediaInterface $media, string $format, strin
             throw new \RuntimeException(sprintf('Cannot use X-Sendfile or X-Accel-Redirect with non %s.', Local::class));
         }
 
+        $directory = $adapter->getDirectory();
+
         $filename = sprintf(
             '%s/%s',
-            $adapter->getDirectory(),
+            false !== $directory ? $directory : '',
```

Comment:
Should we do this or throw an exception ?

Reply:
I think this is the same case, we should throw an exception. I will change those cases.

Reply:
Done.